src/FlightDisplay/PreFlightCheckList.qml

```diff
@@ -42,9 +42,14 @@ Rectangle {
                 group = checkListRepeater.itemAt(index + 1)
                 group.enabled = true
                 group._checked = true
+            } else if ((index + 1) == checkListRepeater.count) {
+                // If the last group is passed, we mark the entire checklist as passed.
+                _passed = true
+                checklistDropPanel.close()
```

Comment:
I'm not so sure about this. There were original problems for example with collapsing groups when they passed. It tends to happen when the last item to pass it not under user control. Things just magically collapse/go away and the user has no idea what just happen. With the time on group passing you can see the last item go green visually and then a short time after it closes. That way the user knows what happened. By closing the drop down right away and last pass you run into the same exact problem. I think it might be ok to auto-close, but if so it needs to work on a delayed timer basis just like the groups. That way you can see it mark as passed, register that in your head and then you see it close.

Reply:
Gotcha. I think now that you can just click away from the check list to minimize it the auto closing is not as important. I removed it.